app/src/main/java/org/xbmc/kore/ui/sections/localfile/LocalFileActivity.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2015 DanhDroid. All rights reserved.
```

Comment:
Your name or XBMC Foundation

Reply:
I think I need to keep the original author as well, as the file has been copied.